clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesDescriptionListGroup.tsx

```diff
@@ -19,120 +13,17 @@ const ModelPropertiesDescriptionListGroup: React.FC<ModelPropertiesDescriptionLi
   customProperties = {},
   isArchive,
   saveEditedCustomProperties,
-}) => {
-  const [editingPropertyKeys, setEditingPropertyKeys] = React.useState<string[]>([]);
```

Comment:
Ah, I hadn't realized we duplicated all this logic here and in ModelPropertiesExpandableSection, nice job deduplicating that.